lib/workers/bin/run_single_worker.rb

```diff
@@ -36,8 +38,8 @@
     exit
   end
 
-  opts.on("-f", "--force", "Ignore missing server roles and run anyway") do
-    options[:force] = true
+  opts.on("-r=ROLE", "--role=ROLE", "Set active role") do |val|
+    (options[:roles] ||= []) << val
```

Comment:
This might need an example.  It's unclear if it supports one, both, or neither of the following:

`-r "ems_inventory,user_interface"`
`-r ems_inventory -r user_interface`

Reply:
do we need to print the list of server role names when they use `-h` ?

Reply:
Yeah good point on how to use --role for multiple will update that.
Is there a nice way to get the list of all the possible roles?

Reply:
Okay updated the description to make it more obvious how it should be called, I switched it to a comma separated list because it is more in-line with `--ems-id` here.

Reply:
> Is there a nice way to get the list of all the possible roles?

At this point, we haven't loaded rails (config/environment) so we can't query the tables or use any of the rails niceties.  

I don't know about nice but this could be thrown into a method that could be called outside of rails...

```ruby
irb(main):006:0> File.read("db/fixtures/server_roles.csv").lines.collect {|line| line.split(",").first}
=> ["name", "automate", "cockpit_ws", "database_operations", "database_owner", "embedded_ansible", "ems_inventory", "ems_metrics_collector", "ems_metrics_coordinator", "ems_metrics_processor", "ems_operations", "event", "git_owner", "internet_connectivity", "notifier", "reporting", "scheduler", "smartproxy", "smartstate", "user_interface", "remote_console", "web_services"]
```

Reply:
:+1: done, and I shifted the first row off so `"name"` isn't there